database/migrations/2025_04_15_155615_create_bills_table.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('bills', function (Blueprint $table) {
+            $table->id();
+            $table->text('description')->nullable();
+            $table->decimal('amount', 10, 2);
+            $table->string('reference_number')->unique();
+            $table->unsignedBigInteger('model_id')->nullable();
+            $table->foreignId('branch_id')->constrained('branches');
```

Comment:
Falta la restriccion de borrado
```suggestion
            $table->foreignId('branch_id')->constrained('branches')->retrictOnDelete();
```